Component/Text-Utilies/text-utilities.css

```diff
@@ -65,4 +101,11 @@ body {
     }
 } 
   */
+/* @media screen and (min-width : 1536px){
+
+    html{
+        font-size:150%;
+    }
+} 
+  */
```

Comment:
this could be cleared

Reply:
i kept them for later stages of making website responsive